.github/pull_request_template.md

```diff
@@ -1,23 +1,38 @@
-**Thank you for submitting this pull request**
-
-**JIRA**: _(please edit the JIRA link if it exists)_ 
-
-[link](https://www.example.com)
-
-**referenced Pull Requests**: _(please edit the URLs of referenced pullrequests if they exist)_
-
-paste the link(s) from GitHub here
-
-**How to retest or run**:
-
-* a pull request please add comment: regex **[.\*[j|J]enkins,?.\*(retest|test) this.\*]**
-
-* a full downstream build please add comment: regex **[.*\[j|J]enkins,?.\*(execute|run|trigger|start|do) fdb.\*]**
-
-* a compile downstream build please  add comment: regex **[.\*[j|J]enkins,?.\*(execute|run|trigger|start|do) cdb.\*]**
-
-* a full production downstream please add comment: regex **[.\*[j|J]enkins,?.\*(execute|run|trigger|start|do) product fdb.\*]**
-
-* an upstream build please add comment: regex **[.\*[j|J]enkins,?.\*(execute|run|trigger|start|do) upstream.\*]**
-
-i.e for running a full downstream build =  **Jenkins do fdb**
+<!--
+Thank you for submitting this pull request.
+
+Please provide all relevant information as outlined below. Feel free to delete
+a section if that type of information is not available.
+-->
+
+### JIRA
+
+<!-- Add a JIRA ticket link if it exists. -->
+<!-- Example: https://issues.redhat.com/browse/PLANNER-1234 -->
+
+### Referenced pull requests
+
+<!-- Add URLs of all referenced pull requests if they exist. This is only required when making
+changes that span multiple kiegroup repositories and depend on each other. -->
+<!-- Example:
+* https://github.com/kiegroup/droolsjbpm-build-bootstrap/pull/1234
+* https://github.com/kiegroup/drools/pull/3000
+* https://github.com/kiegroup/optaplanner/pull/899
+* etc.
+-->
+
+### Checklist
+- [ ] Documentation updated if applicable.
+- [ ] Upgrade recipe provided if applicable.
```

Comment:
Not sure about the checklist. I'm afraid it will be confusing most of the times because it will stay in the description unchecked. In reality both checklist items have 3 states: -1) not applicable, 0) required but not included, 1) included. So I'm afraid that for reviewer it will be confusing whether an empty checkbox means 0 and the PR is blocked on that or it means -1.

Reply:
That's why it's formulated as "*Something* updated if applicable." - if there is no update necessary, it should be checked. Of course, it doesn't guarantee that what you describe cannot happen.

Reply:
We can give it a try and if we see it's a source of confusion, we can always remove the checklist.